src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ActiveRequestsPage/columns.ts

```diff
@@ -0,0 +1,40 @@
+import { DataTableColumn } from '@equinor/fusion-components';
+import PersonnelRequest from '../../../../../../models/PersonnelRequest';
+import RequestStateFlow from '../components/RequestStateFlow';
+
+const columns: DataTableColumn<PersonnelRequest>[] = [
+    {
+        accessor: request => request.position?.basePosition?.name || '',
+        key: 'basePosition',
+        label: 'Base position',
+        sortable: true,
+    },
+
+    {
+        accessor: request => request.person?.name || '',
+        key: 'person',
+        label: 'Person',
+        sortable: true,
+    },
+    {
+        accessor: request => request.state.toString(),
+        key: 'status',
+        label: 'Status',
+        component: RequestStateFlow,
+        sortable: true,
+    },
+    {
+        accessor: request => request.position?.name || '',
+        key: 'position',
+        label: 'Position',
+        sortable: true,
+    },
+    {
+        accessor: request => request.position?.taskOwner?.positionId || '',
```

Comment:
Should this use a position card/column?

Reply:
@MartinForre Moved` PositionColumn` component from `./src/pages/ProjectPage/pages/ContractsOveriewPage/`components to `./src/pages/ProjectPage/components` and used that